includes/payment-methods/class-afterpay-payment-method.php

```diff
@@ -24,12 +24,13 @@ class Afterpay_Payment_Method extends UPE_Payment_Method {
 	 */
 	public function __construct( $token_service ) {
 		parent::__construct( $token_service );
-		$this->stripe_id           = self::PAYMENT_METHOD_STRIPE_ID;
-		$this->title               = __( 'Afterpay', 'woocommerce-payments' );
-		$this->is_reusable         = false;
-		$this->icon_url            = plugins_url( 'assets/images/payment-methods/afterpay.svg', WCPAY_PLUGIN_FILE );
-		$this->currencies          = [ 'USD', 'CAD', 'AUD', 'NZD', 'GBP', 'EUR' ];
-		$this->limits_per_currency = [
+		$this->stripe_id                     = self::PAYMENT_METHOD_STRIPE_ID;
+		$this->title                         = __( 'Afterpay', 'woocommerce-payments' );
+		$this->is_reusable                   = false;
+		$this->icon_url                      = plugins_url( 'assets/images/payment-methods/afterpay.svg', WCPAY_PLUGIN_FILE );
+		$this->currencies                    = [ 'USD', 'CAD', 'AUD', 'NZD', 'GBP', 'EUR' ];
```

Comment:
We an drop 'EUR' as the BNPLs rollout didn't include EU-countries.

Reply:
Given that it's wrongly introduced with the BNPL merge, I'd feel better merging simple one line PR for this so that the reasoning of removing EUR is fully clear (not hidden inside this bug fix).

Reply:
Ok, we can address this in a separate PR.